comictaggerlib/taggerwindow.py

```diff
@@ -214,16 +214,31 @@ def __init__(
 
         if config[0].Runtime_Options__type and isinstance(config[0].Runtime_Options__type[0], str):
             # respect the command line option tag type
-            config[0].internal__save_data_style = config[0].Runtime_Options__type[0]
+            config[0].internal__save_data_style = config[0].Runtime_Options__type
             config[0].internal__load_data_style = config[0].Runtime_Options__type[0]
 
-        if config[0].internal__save_data_style not in metadata_styles:
-            config[0].internal__save_data_style = list(metadata_styles.keys())[0]
+        for style in config[0].internal__save_data_style:
+            if style not in metadata_styles:
+                config[0].internal__save_data_style.remove(style)
         if config[0].internal__load_data_style not in metadata_styles:
             config[0].internal__load_data_style = list(metadata_styles.keys())[0]
-        self.save_data_style = config[0].internal__save_data_style
+        self.save_data_styles = config[0].internal__save_data_style
         self.load_data_style = config[0].internal__load_data_style
 
+        # Add multiselect combobox
+        self.cbSaveDataStyle = CheckableComboBox()
+        self.cbSaveDataStyle.setToolTip("At least 1 save style is required")
+        # Add normal combobox for read style (TODO support multiple read styles)
+        self.cbLoadDataStyle = QtWidgets.QComboBox()
+
+        # Need to set minimum or formLayout will resize larger than 230px (info and cover width)
+        self.cbLoadDataStyle.setMinimumWidth(100)
+        self.cbSaveDataStyle.setMinimumWidth(100)
+        self.formLayout.addRow("Read Style", self.cbLoadDataStyle)
+        self.formLayout.addRow("Modify Styles", self.cbSaveDataStyle)
+        # Gets overwritten from ui file after using addRow
```

Comment:
How does this get overwritten from the ui file? It is loaded before this on line 89